_layouts/base.html

```diff
@@ -10,6 +10,39 @@
         {% if page.description %}<meta name="description" content="{{ page.description }}">{% endif %}
         <link href="{{ site.baseurl }}/css/custom.css" rel="stylesheet" type="text/css">
         <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">
+        <!--Below is the schema annotation for blogposts-->
+        {% if page.layout == 'post' %}
+        <script type="application/ld+json">
+          {
+            "@context": "https://schema.org",
+            "@type": "BlogPosting",
+            "mainEntityOfPage": {
+              "@type": "WebPage",
+              "@id": "{{ site.url }}/posts/"
+              },
+            "headline": "{{ page.title }}",
+            "description": "{{ page.excerpt | strip_html | strip_newlines | truncate: 200 }}",
+            "image": "{{ page.image }}",
+            "author": {
+                  "@type": "Person",
+                  "name": "{{ page.authors | join: ", "}}",
+                  "url": "{{ site.url }}/people#{{ page.authors[0] }}"
+              },
+            "publisher": {
+              "@type": "Organization",
+              "name": "Open Life Science",
+                "logo": {
+                  "@type": "ImageObject",
+                  "url": "{{ site.url }}/images/logo.png"
+                }
+              },
+            "datePublished": "
+              {%- assign date_format = site.minima.date_format | default: "%Y-%m-%d" -%}
+              {{ page.date | date: date_format }}",
```

Comment:
You can probably simplify with only `{{ page.date | date: "%Y-%m-%d" }}`

Reply:
I tried this befor; it didn't work. Totally worked this time. 😆 Thanks!